test/blueprint/entity-blueprint.spec.js

```diff
@@ -99,6 +114,14 @@ describe('JHipster entity generator with blueprint', () => {
             it('contains the specific change added by the blueprint', () => {
                 assert.fileContent(`${CLIENT_MAIN_SRC_DIR}i18n/en/foo.json`, /awesomeAngularAppName/);
             });
+
+            // Verify if the custom entity config is persisted.
+            it('contains the specific config added', () => {
+                assert.fileContent('.jhipster/Foo.json', /customPreConfigValue/);
+            });
+            it('contains the specific config added by the blueprint', () => {
+                assert.fileContent('.jhipster/Foo.json', /customBlueprintConfigValue/);
```

Comment:
```suggestion
                assert.fileContent('.jhipster/Foo.json',  /"customBlueprintConfigKey": "customBlueprintConfigValue"/);
```
Again, to ease future understanding